optuna_dashboard/_serializer.py

```diff
@@ -161,6 +163,10 @@ def serialize_study_detail(
     form_widgets = get_form_widgets_json(system_attrs)
     if form_widgets:
         serialized["form_widgets"] = form_widgets
+    if _SYSTEM_ATTR_FEEDBACK_COMPONENT_TYPE in system_attrs:
+        serialized["feedback_component_type"] = system_attrs[_SYSTEM_ATTR_FEEDBACK_COMPONENT_TYPE]
+    if _SYSTEM_ATTR_FEEDBACK_ARTIFACT_KEY in system_attrs:
+        serialized["feedback_artifact_key"] = system_attrs[_SYSTEM_ATTR_FEEDBACK_ARTIFACT_KEY]
```

Comment:
Could you update these lines like this?

```suggestion
    serialized["preferential_setting"] = system_attrs[_SYSTEM_ATTR_PREFERENCE_...]
```